src/pages/Home/Home.jsx

```diff
@@ -12,11 +13,14 @@ const Home = () => {
   };
 
   return (
+    <>
     <div>
       <h1 className={styles["heading"]}>Tecnoesis 2024</h1>
       <Input value={demoState} onChange={changeDemoState} />
       <Button>Hello</Button>
     </div>
+    <About></About>
```

Comment:
Change it to auto closing tag `<About/>`

Reply:
Doing the changes bhaiya